normits_demand/models/notem/production_models.py

```diff
@@ -329,26 +322,23 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
+                msg= "No code implemented to constrain productions"
+                self._logger.debug(msg)
+                warnings.warn(msg)
```

Comment:
This shouldn't be a warning - as there was an error here already, just raise the error with the message. e.g.

`raise NotImplemented(msg)`

Reply:
updated as per comment